test/contrib/rails/apps/rails3.rb

```diff
@@ -13,3 +14,6 @@ class Rails3 < Rails::Application
 require 'contrib/rails/apps/controllers'
 Rails3.initialize!
 require 'contrib/rails/apps/models'
+
+# TODO[christian]: remove this, it's required for tests to pass, could hide some bug
+require 'contrib/rails/core_extensions'
```

Comment:
Curious, removing this breaks rails3 tests... Curious because this file is required by ddtrace which is required before.